addons/web_editor/static/src/js/editor/odoo-editor/src/commands/commands.js

```diff
@@ -509,15 +509,11 @@ export const editorCommands = {
             !descendants(block).some(descendant => selectedBlocks.includes(descendant))
         ));
         for (const node of deepestSelectedBlocks) {
-            if (node.nodeType === Node.TEXT_NODE && isWhitespace(node) && closestElement(node).isContentEditable) {
-                node.remove();
-            } else {
-                let block = closestBlock(node);
-                if (!['OL', 'UL'].includes(block.tagName) && block.isContentEditable) {
-                    block = block.closest('li') || block;
-                    const ublock = block.closest('ol, ul');
-                    ublock && getListMode(ublock) == mode ? li.add(block) : blocks.add(block);
-                }
+            let block = closestBlock(node);
```

Comment:
Hello @visp-odoo just a simple question , this code is working for saas-16.2 right? If yes then whats changed such that we need to remove those conditions?